java/test/jmri/jmrix/lenz/XNetTrafficControllerTest.java

```diff
@@ -11,24 +27,321 @@
  * @author Paul Bender Copyright (C) 2016
  */
 public class XNetTrafficControllerTest extends jmri.jmrix.AbstractMRTrafficControllerTest {
-
+    XNetTestSimulator testAdapter;
+    XNetPacketizer lnis;
+    
+    volatile boolean blockMessageQueue;
+    final Semaphore messageQueuePermits = new Semaphore(0);
+    volatile boolean timeoutOccured;
+    
+    XNetTurnoutManager xnetManager;
```

Comment:
The tests that use the turnout manager in this file are in the wrong location.  So that they are identifiable as testing something related to the turnout ( they are not unit tests for the traffic controller ) They should either be in the XNetTurnoutManagerTest or the XNetTurnoutTest.  Alternatively, you can add a new test class called XNetTurnoutIT which will identify the tests as integration tests.